components/library/tntnet/tntnet-server.p5m

```diff
@@ -33,5 +33,5 @@ file path=usr/share/man/man7/tntnet.xml.7
 
 file path=usr/share/tntnet/tntnet-conf2xml.pl
 
-file path=usr/etc/init.d/tntnet
-file path=usr/etc/tntnet/tntnet.xml
+file path=etc/init.d/tntnet
```

Comment:
So, I see two options:
a) it can wait for SMF to be created;
b) we don't deliver init.d script (or daemon at all), so that people don't start using it.